tests/frontend/test_ops_gate.py

```diff
@@ -165,6 +165,18 @@ def dummy_apply(self, reg, backend, **kwargs):
         # dagger should negate the first param
         assert applied_params == [-orig_params[0]] + orig_params[1:]
 
+class TestGKPBasics:
+    """Test the basic properties of the GKP state preparation"""
+
+    @pytest.mark.parametrize("state", [[0], [0, 0, 1]])
```

Comment:
Are these allowable values for state? I thought state had to be of the form [theta, phi]

Reply:
I don't think so, but, this tests is just checking that things are printed correctly.

Reply:
Was me adding these values, happy for them to be changed!